spec/config_env_spec.rb

```diff
@@ -22,6 +22,7 @@
       Config.env_separator        = '.'
       Config.env_converter        = :downcase
       Config.env_parse_values     = true
+      Config.env_parse_arrays     = true
```

Comment:
Why this entire block is here? It actually makes specs in this file being unable to test gem's defaults

Reply:
Yeah, I agree that's weird. It'd be better if each test was able to rely on having the defaults set and then be explicit about when they should be different